src/TMap.cpp

```diff
@@ -1074,7 +1074,6 @@ bool TMap::serialize(QDataStream& ofs, int saveVersion)
     ofs << mpRoomDB->getAreaMap().size();
     // serialize area table
     QMapIterator<int, TArea*> itAreaList(mpRoomDB->getAreaMap());
-    QList<int> areasWithData; // TODO: Remove when versions < 17 are not an option
```

Comment:
Suspicious that this is unused - @SlySven ?

Reply:
Nope, a left over from when I had warning code if the user was trying to save a map with Area user data in a map format that didn't support it (i.e. those less than 17) which was issued if this was NOT empty after all the areas were saved - it is safe to remove now.